assignment/core/src/main/java/com/assignment/core/config/RSA_AES_Config.java

```diff
@@ -0,0 +1,26 @@
+package com.assignment.core.config;
+
+
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.AttributeType;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+
+@ObjectClassDefinition(name = "RSA and AES Configuration")
+public @interface RSA_AES_Config {
+    @AttributeDefinition(
+            name = "Public Key" ,
+            type= AttributeType.STRING
+    )
+    String publicKey() default "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCYQYM3FwZuQNLK7xRbuBbA...";
```

Comment:
Why is there a default value for public/private key ? 
Does it make any sense ?